ppa/archive/models.py

```diff
@@ -1030,6 +1060,9 @@ def hathi_page_index_data(cls, digwork):
         # get page span from digitized work
         page_span = digwork.page_span
         digwork_index_id = digwork.index_id()
+        # digwork index id is fallback for cluster, since it is used
+        # to collapse works and pages that belong together
+        cluster_id = str(digwork.cluster) if digwork.cluster else digwork_index_id
```

Comment:
This line makes it such that cluster_id is always set. Mary found this in testing.

Either we need a cluster name or separate out join logic from cluster logic.